view/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
I think that you can delete this file.

Reply:
You will need to write tests at some point tho. Fine to delete for now, but tests are certainly required.